app/Console/Commands/SendIntegrationActivationReminderEmailCommand.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use App\Domain\Integrations\IntegrationType;
+use App\Domain\Integrations\Repositories\IntegrationRepository;
+use App\Mails\MailJet\MailjetConfig;
+use App\Mails\SendIntegrationActivationReminderEmail;
+use Illuminate\Console\Command;
+use Illuminate\Support\Collection;
+
+final class SendIntegrationActivationReminderEmailCommand extends Command
```

Comment:
I think the suffix `Command` is not really needed.

Reply:
It needs to have this because the service it uses it called `SendIntegrationActivationReminderEmail`.
I open to suggestions for another name.

Reply:
Maybe call the service `SendActivationReminder`. This also makes me wonder of there is not too much redundancy in the naming, it is always for `Integration` and it is always an `Email`

Reply:
Renamed class to SearchExpiredIntegrations, and removed the old Service Dependency (like discussed on slack) because it didn't do anything anymore.

The handle() now throws an Event called ActivationExpired and all references to sending an email are gone.

Also add a new test for the new way of working.